kroxylicious-runtime/src/main/java/io/kroxylicious/proxy/KafkaProxy.java

```diff
@@ -125,7 +125,7 @@ public KafkaProxy startup() {
                     .map(c -> new HostPort(c.getEffectiveBindAddress(), c.getEffectivePort()));
             portConflictDefector.validate(virtualClusterModels, managementHostPort);
 
-            var availableCores = Runtime.getRuntime().availableProcessors();
+            int availableCores = config.eventLoopThreadCount().orElse(Runtime.getRuntime().availableProcessors());
```

Comment:
Should the defaulting happen here? Or be pushed down into the configuration record?